bundles/org.openhab.binding.miele/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -6,4 +6,13 @@
 	<name>Miele Binding</name>
 	<description>This is the binding for Miele@home appliances</description>
 
+	<config-description>
+		<parameter name="removalGracePeriod" type="integer" min="0" step="1" unit="s">
+			<label>Removal Grace Period</label>
+			<description>Extra grace period (seconds) that MDns discovery shall wait before removing a lost Bridge from the
```

Comment:
Please add the texts to _src\main\resources\OH-INF\i18n\miele.properties_ also for being translatable.

Reply:
This refers to a binding configuration parameter, where the syntax of the properties file is not explicitly explained in the developer guide, but I guessed it could be as shown below. => Please check and confirm!

```
binding.config.miele.removalGracePeriod.label = 
binding.config.miele.removalGracePeriod.description =
```

Reply:
I checked some other bindings, and it seems the syntax that I guessed is indeed correct.